screens/Authentication/Signin.tsx

```diff
@@ -1,36 +1,80 @@
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
-import { useAuthentication } from "../utils/hooks/useAuthentication";
-import { Button } from "react-native-elements";
+import { StyleSheet, Text, View, Image } from "react-native";
+import { useAuthentication } from "../../utils/hooks/useAuthentication";
+import { Pressable } from "react-native";
 import { getAuth, signOut } from "firebase/auth";
 
 const auth = getAuth();
 
+const logo = require('../../assets/favicon.png');
+
 const SigninScreen = ({ navigation }: any) => {
   const { user } = useAuthentication();
 
   return (
+
+    <View style={styles.logocontainer} >
+    <Image source={logo} style={styles.logo}/>
     <View style={styles.container}>
-      <Text>Sign in</Text>
-      <Button
-        title="Back"
+      
+      <Text style={styles.text1}>Welcome to the Chinese Newcomers Service Center job portal!</Text>
+      <Text style={styles.text2}>Let's get your account set up.</Text>
+     
+      <Pressable
         style={styles.button}
         onPress={() => navigation.navigate("Home")}
-      />
+      >
+        <Text style={styles.text3}> SIGN UP </Text>
+      </Pressable>
+      
+      
+
+    </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
```

Comment:
Create a new `style.tsx` file in the Authentication folder and move styling there.